iati-organisations-schema.xsd

```diff
@@ -104,7 +104,7 @@
         <xsd:annotation>
           <xsd:documentation xml:lang="en">
             Default ISO 4217 currency code for all financial values in
-            this activity report. If this is not declared then the
+            this organisation report. If this is not declared then the
```

Comment:
Agree - the org file contains financial data, so this should say "organisation".